### PR TITLE
Cilium Handover

### PR DESCRIPTION
**Pull request by T-Kukawka**
Unsure how to change this: https://github.com/giantswarm/prometheus-rules/blob/master/helm/prometheus-rules/templates/recording-rules/service-level.rules.yml

### CODE REVIEW

File: helm/prometheus-rules/templates/recording-rules/service-level.rules.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ spec:
         label_application_giantswarm_io_team: {{ include "providerTeam" . }}
       record: slo_target
 
-      # -- daemonset
+      # -- KAAS daemonset
     - expr: |
         label_replace(
           kube_daemonset_status_desired_number_scheduled{namespace=~"giantswarm|kube-system", daemonset=~"aws-node|aws-cloud-controller-manager|ebs-csi-node|calico-node|cert-exporter|kube-proxy|net-exporter|node-exporter|azure-cloud-controller-manager|azure-cloud-node-manager|azure-scheduled-events|csi-azuredisk-node|cilium"},
```
Comment (mcharriere) [inline helm/prometheus-rules/templates/recording-rules/service-level.rules.yml:47]: `cilium--`

**Review by QuentinBisson — APPROVED** [review]

File: helm/prometheus-rules/templates/alerting-rules/cilium.rules.yml [hunk 2]
```diff
@@ -17,10 +17,10 @@ spec:
       expr: avg(cilium_bpf_map_pressure) by (cluster_id, map_name) * 100 > 80
       for: 15m
       labels:
-        area: kaas
+        area: managedservices
```
Comment (QuentinBisson) [inline helm/prometheus-rules/templates/alerting-rules/cilium.rules.yml:20]: Is that not platform? I'm confused

Comment (T-Kukawka) [reply helm/prometheus-rules/templates/alerting-rules/cilium.rules.yml:20]: i based this on other alerts and there were `managedservices` used

Comment (QuentinBisson) [reply helm/prometheus-rules/templates/alerting-rules/cilium.rules.yml:20]: I do not think you're wrong, we need to sort those labels :)

**Pull request merged**